core/src/main/java/io/questdb/griffin/SqlExecutionContextImpl.java

```diff
@@ -288,6 +294,11 @@ public void setJitMode(int jitMode) {
         this.jitMode = jitMode;
     }
 
+    @Override
+    public void setJitUsed(boolean value) {
+        jitUsed = true;
```

Comment:
```suggestion
        jitUsed = value;
```

Reply:
not needed, fixed here https://github.com/questdb/questdb/pull/4632/commits/f6400def4728be8e8270a8a2f8d0b1171942ff2a